public void testInnerList() throws Exception
